src/services/__mocks__/diaryApi.js

```diff
@@ -0,0 +1,3 @@
+export const setItem = () => null
```

Comment:
Doesn't block the PR, but I would just set them as `jest.fn()` here, then you can just use them and expect().toBeCalled().

Reply:
Nice.